layers/stateless/sl_cmd_buffer.cpp

```diff
@@ -227,7 +227,7 @@ bool StatelessValidation::ValidateCmdBindVertexBuffers2(VkCommandBuffer commandB
                     not_null_msg = "pSizes is not NULL";
                 else
                     not_null_msg = "pStrides is not NULL";
-                const char *vuid_breach_msg = "%s: %s, so bindingCount must be greater that 0.";
+                const char *vuid_breach_msg = "%s: %s, so bindingCount must be greater than 0.";
                 skip |= LogError(commandBuffer, "VUID-vkCmdBindVertexBuffers2-bindingCount-arraylength", vuid_breach_msg, api_call,
```

Comment:
Don't see much point to the `vuid_breach_msg` variable.

```suggestion
                skip |= LogError(commandBuffer, "VUID-vkCmdBindVertexBuffers2-bindingCount-arraylength", "%s: %s, so bindingCount must be greater than 0.", api_call,
```

Reply:
agree, let me fix this while here

Reply:
I only found this because I mis-spelled `than` and noticed this code lol